docs/setup/import-mvcc.md

```diff
@@ -93,7 +93,7 @@ lakefs import --with-merge lakefs://example-repo -m s3://example-bucket/path/to/
 
 1. You can download the `lakefs` binary from [here](https://github.com/treeverse/lakeFS/releases). Make sure you choose one compatible with your installation of lakeFS.
 
-1. Use a configuration file like the one used to start your lakeFS installation. This will be used to access your database. An example can be found [here](http://localhost:4000/reference/configuration.html#example-aws-deployment).
+1. Use a configuration file like the one used to start your lakeFS installation. This will be used to access your database. An example can be found [here](../reference/configuration.html#example-aws-deployment).
```

Comment:
```suggestion
1. Use a configuration file like the one used to start your lakeFS installation. This will be used to access your database. An example can be found [here](../reference/configuration.md#example-aws-deployment).
```
(but I vote to delete this page altogether)

Reply:
Yeah - there's one reference on the import page - but I think we can let it go

Reply:
"The snow glows white on the mountain tonight
Not a footprint to be seen
A kingdom of isolation
And it looks like I'm the queen"